modules/Bio/EnsEMBL/ArchiveStableId.pm

```diff
@@ -529,5 +535,17 @@ sub successors {
   return $self->{'successors'};
 }
 
+sub meta_value {
+  my $self = shift;
+  $self->{meta_value} = shift if (@_);
+  return $self->{'meta_value'};
+}
+
+sub species_id {
+  my $self = shift;
+  $self->{species_id} = shift if (@_);
+  return $self->{'species_id'};
+}
+
```

Comment:
should be inherited from BaseAdaptor, see code in SliceAdaptor (eg https://github.com/Ensembl/ensembl/blob/release/89/modules/Bio/EnsEMBL/DBSQL/SliceAdaptor.pm#L261)